extensions/binding/org.eclipse.smarthome.binding.weatherunderground/ESH-INF/i18n/weatherunderground.properties

```diff
@@ -7,3 +7,4 @@ offline.conf-error-missing-apikey = The "API key" parameter must be configured.
 offline.conf-error-missing-location = The "location" parameter must be configured.
 offline.conf-error-syntax-language = The value of the "language" parameter must contain 2 letters.
 offline.conf-error-min-refresh = The minimum value of the "refresh interval" parameter must be 5 minutes.
+offline.uri-error = Could not create URI to fetch data, please check if parameters are correct and otherwise report a bug.
```

Comment:
- "and otherwise report a bug" is a pretty unusual (and rather inappropriate) message for a Thing status.
- "please check if parameters are correct" - can't we figure out ourselves if a parameter is invalid and if so, which and provide more details here?